app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -21,16 +21,20 @@ import timber.log.Timber
 
 class EventDetailsFragment : Fragment() {
     val EVENT_ID = "EVENT_ID"
+    val EVENT_IDENTIFIER = "EVENT_IDENTIFIER"
```

Comment:
you can get identifier from event_id , why this is needed

Reply:
Oops really ?

Reply:
@simarsingh24 Nope we cant. id and identifier are both different

Reply:
you already have the event why cant yo get the identifier

Reply:
they are different but having one we can get the other one AFAIK

Reply:
Okay let me just check for a better logic :+1: